docs/general/decentralized-futures.md

```diff
@@ -0,0 +1,45 @@
+---
+id: decentralized-futures
+title: Decentralized Futures Programme
+sidebar_label: Decentralized Futures
+description: Learn about the Decentralized Futures Program.
+keywords: [community, ecosystem, funding]
+slug: ../decentralized-futures
+---
+
+The new [Decentralized Futures Program](https://futures.web3.foundation/) supplements the Grants
+Program, and (as the name suggests) it fits within the recent decentralization efforts around the
+Polkadot ecosystem.
+
+## Program Goal
+
+The goal of the Decentralized Futures Program is to kickstart economically independent, active
+participants in the Polkadot ecosystem.
+
+## Applications
+
+To apply for funding to the Decentralized Futures fill
+[**this form**](https://docs.google.com/forms/d/e/1FAIpQLSdlPyHj56tY6t-5zhdn55VvznR5OACvXe8K1M6vRGmhdslZkw/viewform).
+The form will guide you through each step of the application. Proposals have to be **submitted
+before July 1st, 2024**. Applications will be **reviewed from Dec 1st, 2023 - Jul 31st, 2024** on a
+rolling basis.
+
+## Selection Criteria
+
+In the current phase, we intentionally maintain a level of openness in our approach. This deliberate
+choice stems from the recognition that certain initiatives demand substantial and bold efforts,
+often incurring significant costs, particularly in areas where traditional go-to-market functions
+supported by Parity Technologies are no longer in play. Conversely, there are instances where
+individuals require relatively modest funding in the tens of thousands to initiate their projects.
+This broad approach serves to avoid constraining people's creativity and allows for a wide range of
+possibilities. However, it does come with the drawback of prolonged application processing times.
+
+Several factors significantly enhance the likelihood of application success: Firstly, a demonstrated
```

Comment:
I wouldn't put the dot part first. The two points are more important and there might be cases were teams simply can not take dots, because they have bills to pay. Otherwise looks pretty good